schemas/environmentsSchema.json

```diff
@@ -32,6 +32,15 @@
 					"sshLogin": {
 						"type": "string",
 						"description": "The login used to ssh into the Profile Server."
+					},
+					"serverType": {
+						"type": "string",
+						"description": "The server type. Default value is \"SCA$IBS\"."
+					},
+					"encoding": {
+						"type": "string",
+						"description": "Encoding to use in request/response. The default value is \"utf8\".",
+						"enum": ["ascii", "utf8", "utf16le", "ucs2", "base64", "latin1", "binary", "hex"]
```

Comment:
This list should contain the list of encodings supported on both sides of the connection. As for example, utf16le is not valid on Profile Host.
UTF-8, Latin1 (ISO-8859-1 in JDBC) and ISO-8859-2 (latin2 ?) are supported.
I'll not go with UTF16 as it's very tricky. Please refer to http://tinco.pair.com/bhaskar/gtm/doc/books/pg/UNIX_manual/ch02s08.html to get more details.

Reply:
This list comes from node supported encodings. I will remove utf16 but might as well keep the rest? I will not be able to handle something GTM supports but node does not...